src/packager.ts

```diff
@@ -238,3 +241,12 @@ const objectExists = async (client: S3Client, bucket: string, key: string) => {
     throw error;
   }
 };
+
+const clearTempFolders = async () => {
```

Comment:
I think making this optional, while always cleaning up our own `tempDir`, is a good approach to go with.